### PR TITLE
aws REST address update

### PR DESCRIPTION
**Pull request by alevinru**
## Summary
update AWS RestBaseUrl domain 

## Additional Information
[OKX API Notice](https://www.okx.com/docs-v5/log_en/#upcoming-changes-notice-regarding-the-decommissioning-of-old-okx-domain-names)

### CODE REVIEW

File: src/util/requestUtils.ts [hunk 1]
```diff
@@ -47,7 +47,7 @@ export function getRestBaseUrl(
       return 'https://www.okx.com';
     }
     case 'aws': {
-      return 'https://aws.okex.com';
+      return 'https://aws.oke.com';
```
Comment (tiagosiebler) [inline src/util/requestUtils.ts:50]:
```suggestion
      return 'https://aws.okx.com';
```

**Review by tiagosiebler — CHANGES_REQUESTED** [review]
Thanks for the PR! Small typo

**Comment by alevinru** [comment]
OMG what a shame on me :)

**Review by tiagosiebler — APPROVED** [review]
It happens 😁 thanks again for the PR

**Pull request merged**